package.json

```diff
@@ -9,20 +9,20 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@clerk/nextjs": "^5.4.1",
-    "next": "14.2.8",
+    "@clerk/nextjs": "^6.1.0",
+    "next": "15.0.2",
```

Comment:
next@15 requires react@19. They have an upgrade command that will update all of the necessary spots. 

`npx @next/codemod@latest upgrade latest`

Reply:
Their codemod didn't upgrade React, but I did that manually.